bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,163 @@
+    createOption("Vermont", state);
+    createOption("Virginia", state);
+    createOption("Washington", state);
+    createOption("West Virginia", state);
+    createOption("Wisconsin", state);
+    createOption("Wyoming", state);
+
+    stateDiv.appendChild(state);
+    addressInputDiv.appendChild(stateDiv);
+    
+    //Zip Code
+    createTextInputElement("Postal", document.getElementById("address-input-div"));
+}
+
+function loadCanadianAddress() {
+    //Address line
+    let addressInputDiv = document.getElementById("address-input-div");
+
+    createTextInputElement("Address Line 1", document.getElementById("address-input-div"));
+    createTextInputElement("Address Line 2", document.getElementById("address-input-div"));
+
+    //City
+    createTextInputElement("City", document.getElementById("address-input-div"));
+
+    //Province
+    let provinceDiv = document.createElement("div");
+    let province = document.createElement("Select"); 
+    province.setAttribute("id", "province-input"); 
+    province.setAttribute("name", "province-input"); 
+    let provinceLabel = document.createElement('label');
+    provinceLabel.setAttribute("for", "state-input");
+    let provinceLabelText = document.createTextNode("Province: ");
+    provinceLabel.appendChild(provinceLabelText);
+    provinceDiv.appendChild(provinceLabel);
+
+    createOption("Alberta", province);
+    createOption("British Columbia", province);
+    createOption("Manitoba", province);
+    createOption("Alberta", province);
+    createOption("New Brunswick", province);
+    createOption("Newfoundland and Labrador", province);
+    createOption("Northwest Territories", province);
+    createOption("Nova Scotia", province);
+    createOption("Nunavut", province);
+    createOption("Ontario", province);
+    createOption("Prince Edward Island", province);
+    createOption("Quebec", province);
+    createOption("Saskatchewan", province);
+    createOption("Yukon", province);
+
+    provinceDiv.appendChild(province);
+    addressInputDiv.appendChild(provinceDiv);
+
+    //Postal Code
+    createTextInputElement("Postal", document.getElementById("address-input-div"));
+    
+}
+
+function createOption(optionValue, selectToAppend) {
+    let option = document.createElement("option");
+    option.setAttribute("value", optionValue)
+    let optionText = document.createTextNode(optionValue);
+    option.appendChild(optionText);
+    selectToAppend.appendChild(option);
+}
+
+function createTextInputElement(inputElement, divToAppend) {
+    let inputDiv = document.createElement("div");
+    inputDiv.setAttribute("id", (inputElement + "-div"));
+    let inputLabel = document.createElement('label');
+    inputLabel.setAttribute("for", (inputElement + "-label"));
+    let inputLabelText = document.createTextNode(inputElement + ": ");
+    inputLabel.appendChild(inputLabelText);
+    inputDiv.appendChild(inputLabel);
+    let input = document.createElement("input");
```

Comment:
I'm pretty sure all of these lets can be consts..

Reply:
Made const